net/radsecproxy/src/opnsense/mvc/app/models/OPNsense/RadSecProxy/RadSecProxy.xml

```diff
@@ -0,0 +1,372 @@
+                    <Required>N</Required>
+                    <Constraints>
+                        <check001>
+                            <ValidationMessage>Must be set for UDP-clients.</ValidationMessage>
+                            <type>SetIfConstraint</type>
+                            <field>type</field>
+                            <check>udp</check>
+                        </check001>
+                        <check002>
+                            <ValidationMessage>Must be set for TCP-clients.</ValidationMessage>
+                            <type>SetIfConstraint</type>
+                            <field>type</field>
+                            <check>tcp</check>
+                        </check002>
+                    </Constraints>
+                </secret>
+
+                <tlsConfig type="ModelRelationField">
+                    <Required>N</Required>
+                    <Model>
+                        <tlsConfigs>
+                            <source>OPNsense.RadSecProxy.RadSecProxy</source>
+                            <items>tlsConfigs.tlsConfig</items>
+                            <display>name</display>
+                        </tlsConfigs>
+                    </Model>
+                </tlsConfig>
+
+                <certificateNameCheck type="OptionField">
+                    <Required>Y</Required>
+                    <default>off</default>
+                    <OptionValues>
+                        <on>On</on>
+                        <off>Off</off>
+                    </OptionValues>
+                </certificateNameCheck>
+
+                <matchCertificateAttribute type="TextField">
+                    <Required>N</Required>
+                </matchCertificateAttribute>
+
+                <customConfig type="TextField">
+                    <Required>N</Required>
+                </customConfig>
+            </client>
+        </clients>
+
+        <servers>
+            <server type="ArrayField">
+
+                <identifier type="TextField">
+                    <Required>Y</Required>
+                    <mask>/^([0-9a-zA-Z_\-]){1,25}$/u</mask>
+                    <ValidationMessage>Should be a string between 1 and 25 characters whithout special characters.</ValidationMessage>
+
+                    <Constraints>
+                        <check001>
+                            <type>UniqueConstraint</type>
+                            <ValidationMessage>Identifier already in use</ValidationMessage>
+                        </check001>
+                    </Constraints>
+                </identifier>
+
+                <description type="TextField">
+                    <Required>N</Required>
+                </description>
+                
+                <host type="TextField">
+                    <Required>Y</Required>
+                    <IpAllowed>Y</IpAllowed>
+                </host>
+
+                <port type="PortField">
+                </port>

```

Comment:
This looks a bit naked :) Required? Default?

Reply:
Added "not required", defaults set by radsecproxy.